src/app/forms/templates/general/adr.template.ts

```diff
@@ -649,7 +649,7 @@ export const AdrTemplate: FormNode = {
       value: null,
       type: FormNodeTypes.CONTROL,
       width: FormNodeWidth.L,
-      groups: ['battery_list_applicable', 'battery_list_hide', 'dangerous_goods'],
+      groups: ['battery_list_applicable', 'battery_list_hide'],
```

Comment:
Not sure if its this simple, if you click yes on battery list applicable, then click 'no' on dangerous goods then the reference number will still be shown when its meant to be hidden.

Reply:
Ahhhh, you are right. Nice shout out. I didn't realize the test scenario you mentioned before.